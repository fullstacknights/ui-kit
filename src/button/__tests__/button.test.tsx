import * as React from "react";
import * as renderer from "react-test-renderer";

import { Button } from "../index";

describe("Button test suite", () => {
  test("Snapshot test", () => {
    const tree = renderer.create(<Button>Facebook</Button>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
