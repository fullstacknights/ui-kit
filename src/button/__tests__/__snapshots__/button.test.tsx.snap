// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test suite Snapshot test 1`] = `
<button
  className="sc-bdVaJa igACHl"
  disabled={false}
>
  Facebook
</button>
`;
